Pod/Swiftline/Args/Args.swift

```diff
@@ -29,20 +29,26 @@ public class Args {
       return result
     }
     
-    var result = [String: String]()
-    let parsedArges = ArgsParser.parseFlags(all)
+    var parsedFlags = [String: String]()
+    let parsedArgs = ArgsParser.parseFlags(all)
     
-    parsedArges.0.forEach {
-      result[$0.argument.name] = $0.value ?? ""
+    parsedArgs.0.forEach {
+      parsedFlags[$0.argument.name] = $0.value ?? ""
     }
-    
-    cachedResults = ParsedArgs(flags: result, parameters: parsedArges.1)
+
+    let commandName = parsedArgs.1.first!
+    var arguments = parsedArgs.1
```

Comment:
couple things about these 2 lines: 

seems like you should get the `arguments` var first, then get the `commandName` from there maybe? also, is it reliable to force-unwrap that optional, or should you `guard let` it to make sure its there?